io.github.q962.ClipboardServer.yml

```diff
@@ -0,0 +1,143 @@
+id: io.github.q962.ClipboardServer
+runtime: org.gnome.Platform
+runtime-version: "48"
+sdk: org.gnome.Sdk
+command: ClipboaredServer
+
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --device=dri
+  - --share=ipc
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node22
+build-options:
+  append-path: /usr/lib/sdk/node22/bin
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /tmp
+
+modules:
+  - name: xmake
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/xmake-io/xmake.git
+        tag: v2.9.8
+    cleanup:
+      - /bin
+      - /share
+
+  - name: luajit
+    buildsystem: simple
+    build-commands:
+      - make PREFIX=/app
+      - make PREFIX=/app install
+    sources:
+      - type: git
+        url: https://github.com/LuaJIT/LuaJIT.git
+        tag: v2.1
+    cleanup:
+      - /bin
+      - /lib
+      - /share
+
+  - name: penlight
+    sources:
+      - type: archive
+        url: https://github.com/luarocks/luarocks/archive/refs/tags/v3.11.1.zip
+        strip-components: 1
+        sha256: 698444ba69cfb7b8933aaf97ccddcf61c52bc84d585eb0fef5de02a185af40ff
+      - type: file
+        url: https://luarocks.org/manifests/hisham/luafilesystem-1.8.0-1.src.rock
+        sha256: 576270a55752894254c2cba0d49d73595d37ec4ea8a75e557fdae7aff80e19cf
+      - type: file
+        url: https://luarocks.org/manifests/tieske/penlight-1.14.0-2.src.rock
+        sha256: f36affa14fb43e208a59f2e96d214f774b957bcd05d9c07ec52b39eac7f4a05d
+    post-install:
+      - |
+        mkdir -p /app/tmp/luatree
+        luarocks config rocks_trees\[2\].root /app/tmp/luatree
+        luarocks config local_cache /tmp
+
+        luarocks install luafilesystem-1.8.0-1.src.rock
+        luarocks install penlight-1.14.0-2.src.rock
+    cleanup:
+      - /bin
+      - /etc
+      - /share
+
+  - name: dnssd
+    buildsystem: simple
+    build-commands:
+      - chmod u+x dnssd
+      - cp dnssd /app/
+    sources:
+      - type: file
+        url: https://github.com/q962/dnssd/releases/download/v1.2.15/dnssd-Linux
+        sha256: de5289660b45e3cf2a2e6f903e35b1c181d5ceb07ea049c3a0a0000c016e6faf
+        dest-filename: dnssd
+    cleanup:
+      - /
+
+  - name: luvit
+    buildsystem: simple
+    build-options:
+      # bundle: [elf][zip]
+      no-debuginfo: true
+    build-commands:
+      - |
+        chmod +x luvi
+        cp luvi luvit.zip /app
+    sources:
+      - type: file
+        url: https://github.com/luvit/luvi/releases/download/v2.15.0/luvi-Linux-x86_64-luajit-regular
```

Comment:
why making this x86_64 only? Why not building it from source?

Reply:
there seem to be a binary for aarch64 as well.

Reply:
https://github.com/luvit/luvi/releases/download/v2.15.0/luvi-Linux-aarch64-luajit-regular